### DISCUSSION TITLE
Rust Lsp not running

### DISCUSSION THREAD

**Discussion by sgj123456** [helix-editor/helix#7638] [Troubleshooting]
I am on Arch Linux,
I used the command to check that the environment was working properly, but when I opened a rust project using Helix, the bottom left corner displayed "Language server exited". When I tried to restart LSP, it displayed "lsp not running for the current document"

**Comment by gabydd** [comment]
It would be helpful to see what the logs are showing to help debug this, see https://github.com/helix-editor/helix/wiki/FAQ#access-the-log-file
